packages/clerk-js/src/ui/styledSystem/types.ts

```diff
@@ -1,4 +1,6 @@
 // eslint-disable-next-line no-restricted-imports
+// eslint-disable-next-line no-restricted-imports
+// eslint-disable-next-line no-restricted-imports
```

Comment:
@desiprisg ❓ do we need this two times?

Reply:
Probably did this by accident. Will fix.